app/views/spree/order_mailer/cancel_email.html.haml

```diff
@@ -1,11 +1,29 @@
-%h3
-  = t(".customer_greeting", name: @order.bill_address.firstname)
-%h4
-  = t(".instructions")
-%span.clear
+%table.social.white-bg{width: "100%"}
+  %table.column{align: "left"}
+    %tr
+      %td
+        %h3
+          = t(".customer_greeting", name: @order.bill_address.firstname)
+        %h4
+          = t(".instructions_html", distributor: @order.distributor.name )
+        %img.float-left{src: "#{@order.distributor.logo.url(:medium)}"}
```

Comment:
Do you know where this `float-left` class is defined? I think we have one called `left` that adds `float: left`, but I;m not sure `float-left` exists.

Reply:
Now I can't find in the project where it is defined, but I can create a css selector with this property. What do you think?

Reply:
Looking at the design in the issue, the logo is not floating. I think that you can just remove it.
```suggestion
        %img{src: "#{@order.distributor.logo.url(:medium)}"}
```

Reply:
Alright, @mkllnk I will commit the changes.